src/daemons/rhsmcertd-worker.py

```diff
@@ -42,10 +43,19 @@
 _ = gettext.gettext
 
 
+def exit_on_signal(*args):
+    sys.exit(0)
```

Comment:
How about `_signumber, _stackframe`?  That makes it a little more obvious what they are and that we are not using them.